src/components/block/api.ts

```diff
@@ -0,0 +1,110 @@
+import Block from './index';
+import { BlockToolData, ToolConfig } from '../../../types/tools';
+import { SavedData } from '../../types-internal/block-data';
+import { BlockAPI as BlockAPIInterface } from '../../../types/api';
+
+/**
+ * @param block
+ */
+function BlockAPI(block: Block) {
+  const blockAPI: BlockAPIInterface = {
+    /**
+     * Tool name
+     *
+     * @returns {string}
+     */
+    get name(): string {
+      return block.name;
+    },
+
+    /**
+     * Tool config passed on Editor's initialization
+     *
+     * @returns {ToolConfig}
+     */
+    get config(): ToolConfig {
```

Comment:
There can be a misunderstanding with existed [Tool config](https://editorjs.io/provide-custom-configuration) term.

Reply:
That's exactly the ToolConfig object